modules/server_new/src/test/scala/observe/server/engine/HandleSpec.scala

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2016-2025 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package observe.server.engine
+
+import cats.FlatMap
+import cats.Id
+import cats.Monad
+import cats.data.StateT
+import cats.kernel.Eq
+import cats.laws.discipline.*
+import cats.laws.discipline.arbitrary.*
+import cats.laws.discipline.eq.*
+import cats.syntax.all.*
+import cats.~>
+import fs2.Compiler
+import observe.engine.Handle
+import org.scalacheck.Arbitrary
+import org.scalacheck.Arbitrary.arbitrary
+import org.scalacheck.Cogen
+
+import scala.util.Try
+
+
+class HandleSpec extends munit.DisciplineSuite {
+  given [F[_]: Monad, S, E, O](using
```

Comment:
Probably not needed but should these be on its own arb class?

Reply:
I thought about it and decided that we can split it if we ever need it.